File: Day14/ScrollViewObjC/ScrollViewObjC/SimpleZoom.m
//
//  SimpleZoom.m
//  ScrollViewObjC
//
//  Created by techmaster on 8/21/14.
//  Copyright (c) 2014 Techmaster. All rights reserved.
//

#import "SimpleZoom.h"

@interface SimpleZoom () <UIScrollViewDelegate>
@property (weak, nonatomic) IBOutlet UIScrollView *scrollView;
@property (weak, nonatomic) IBOutlet UIImageView *photo;

@end

@implementation SimpleZoom


- (void)viewDidLoad
{
    [super viewDidLoad];
    self.edgesForExtendedLayout = UIRectEdgeNone;
    self.scrollView.minimumZoomScale = 0.25;
    self.scrollView.maximumZoomScale = 4.0;
    
    
}
- (UIView *)viewForZoomingInScrollView:(UIScrollView *)scrollView
{
    return self.photo;
}

@end

File: Day14/ScrollInSwift/SimpleZoom.swift
//
//  SimpleZoom.swift
//  ScrollInSwift
//
//  Created by Trung on 8/25/14.
//  Copyright (c) 2014 vn.Techmsater. All rights reserved.
//
import UIKit
import Foundation

class SimpleZoom: UIViewController, UIScrollViewDelegate {

    @IBOutlet weak var scrollView: UIScrollView!
    
    @IBOutlet weak var imageView: UIImageView!
    
    override func viewDidLoad() {
        scrollView.minimumZoomScale = 0.25
        scrollView.maximumZoomScale = 4.0
    }
    
    
    func viewForZoomingInScrollView(scrollView: UIScrollView!) -> UIView! {
        return imageView
    }
    
    func scrollViewDidZoom(scrollView: UIScrollView) {
       //println(NSString(format: "%2.2f", Double(scrollView.zoomScale)))
       println("\(scrollView.contentSize.width) - \(scrollView.contentSize.height)")
    }
    
    override func didReceiveMemoryWarning() {
        
    }
}
